### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Rifat8080**
- Added popup modal
- added new css style file for modal css
- added javascript for modal functioning
- added linters properly

### CODE REVIEW

File: popup.css [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    border: none;
+    align-self: center;
+    background: var(--color-primary);
+    font-weight: 500;
+    font-size: 16px;
+    letter-spacing: 2px;
+    padding: 1em 1.5em;
+    color: white;
+    border-radius: 3px;
+  }
```
Comment (AbelG101) [inline popup.css:265]: - [ ] Please make sure you follow the given Figma design when implementing the button styling here as the width of the button is causing the text to break. please make sure to fix that. ✔
- Here's how it looks:
  image
- [ ] Also please make sure you implement the `hover` & `active` states of the button as shown in the below Figma design 👇
  firefox_kz6PEcoelc

File: popup.css [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+
+/* Modal Content/Box */
+.modal-content1 {
+  background-color: #fefefe;
+  margin: 15% auto; /* 15% from the top and centered */
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%; /* Could be more or less, depending on screen size */
+  border-radius: 8px;
+}
```
Comment (AbelG101) [inline popup.css:22]: - [ ] [OTPIONAL] It'd be better if you can center the popup vertically to the screen for a better UX 
- Here's how it looks:
  image

File: index.html [hunk 3]
```diff
@@ -679,7 +288,45 @@ <h1 class="foot-title">
...
+            <span>See Live</span>
+            <object data="./img/icons/external.svg"></object>
+          </a>
+          <a href="" class="popup__link popup__link--code">
+            <span>See Source</span>
+            <object data="./img/icons/github-white.svg"></object>
+          </a>
+        </div>
+      </div>
+    </div> -->
```
Comment (franclobo) [inline index.html:327]: - [ ] Well done using JavaScript to populate this section as requested by the previous CR. Please 🙏🏽 remove the unused code for clean code and good practices of HTML & CSS, we use the comments to describe what the code does to help other programmers to read our code.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Rifat8080, 

# ♻️ Required Changes ♻️

soClose

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️
- Descriptive PR ✔️
- Correct Github flow ✔️ 
- Dynamic webpage ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @Rifat8080 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**